substrate/frame/staking-async/runtimes/papi-tests/tests/common.ts

```diff
@@ -0,0 +1,113 @@
+import { Chain, Observe } from "../src/test-case";
+import { logger, nullifySigned, nullifyUnsigned, type ApiDeclerations } from "../src/utils";
+
+export function commonUnsignedSteps(
```

Comment:
Could you give this (and `commonSigned`) a top-level comment describing what this does?
Just a brief overview of what this function does, what it expects to see at each step, and a termination condition (which probably relates to the args below).

If I may given an example of my own: something like https://github.com/open-web3-stack/polkadot-ecosystem-tests/blob/8acdd828e08dffa37c0c723c806d9028db020ae7/packages/shared/src/scheduler.ts#L536-L543